ion_functions/qc/qartod_functions.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env python
+
+"""
+@package ion_functions.qartod_functions
+@file ion_functions/qartod_functions.py
+@author Seth Foote
+@brief Copy of QARTOD QC functions from ioos qartod repository:
+https://github.com/ioos/qartod/blob/master/ioos_qartod/qc_tests/qc.py#L83
+"""
+import numpy as np
+import pyproj
+import quantities as pq
+import pandas as pd
+import multiprocessing
+
+
+class QCFlags:
+    """Primary flags for QARTOD."""
+    # Don't subclass Enum since values don't fit nicely into a numpy array.
+    GOOD_DATA = 1
+    UNKNOWN = 2
+    SUSPECT = 3
+    BAD_DATA = 4
+    MISSING = 9
```

Comment:
are you sure these flags are correct? I would think they would be: 1, 2, 4, 8, 16 (0b1, 0b10, 0b100, 0b1000, 0b10000). Otherwise, `GOOD_DATA & UNKNOWN == SUSPECT`.